heka/map.jinja

```diff
@@ -10,10 +10,6 @@ Debian:
   - syslog
   service:
   - heka
-  filter: {}
-  splitter: {}
-  encoder: {}
-  decoder: {}
```

Comment:
I am not sure we should remove this, for backward compatibility reasons. @simonpasquier @cznewt?